lib/SILOptimizer/Transforms/CopyForwarding.cpp

```diff
@@ -171,10 +171,13 @@ static SILArgumentConvention getAddressArgConvention(ApplyInst *Apply,
 /// If the given instruction is a store, return the stored value.
 static SILValue getStoredValue(SILInstruction *I) {
   switch (I->getKind()) {
+#define NEVER_LOADABLE_CHECKED_REF_STORAGE(Name, ...) \
+  case SILInstructionKind::Store##Name##Inst:
+#define SOMETIMES_LOADABLE_CHECKED_REF_STORAGE(Name, ...) \
+  case SILInstructionKind::Store##Name##Inst:
+#include "swift/AST/ReferenceStorage.def"
```

Comment:
Again, surely this is all checked ownerships.

Reply:
I've fixed this in a forthcoming pull request update.